cmd/go-filecoin/miner_daemon_test.go

```diff
@@ -263,54 +260,55 @@ func requireMinerCreate(ctx context.Context, t *testing.T, env *fastesting.TestE
 func TestMinerCreateChargesGas(t *testing.T) {
 	t.Skip("Long term solution: #3642")
 	tf.IntegrationTest(t)
-
-	miningMinerOwnerAddr, err := address.NewFromString(fixtures.TestAddresses[0])
-	require.NoError(t, err)
-
-	d1 := makeTestDaemonWithMinerAndStart(t)
-	defer d1.ShutdownSuccess()
-	d := th.NewDaemon(t, th.KeyFile(fixtures.KeyFilePaths()[2])).Start()
-	defer d.ShutdownSuccess()
-	d1.ConnectSuccess(d)
-	var wg sync.WaitGroup
-
-	// make sure the FIL shows up in the MinerOwnerAccount
-	startingBalance := queryBalance(t, d, miningMinerOwnerAddr)
-
-	wg.Add(1)
-	go func() {
-		testMiner := d.RunSuccess("miner", "create", "--from", fixtures.TestAddresses[2], "--gas-price", "333", "--gas-limit", "100", "200")
-		addr, err := address.NewFromString(strings.Trim(testMiner.ReadStdout(), "\n"))
-		assert.NoError(t, err)
-		assert.NotEqual(t, addr, address.Undef)
-		wg.Done()
-	}()
-	// ensure mining runs after the command in our goroutine
-	d1.MineAndPropagate(time.Second, d)
-	wg.Wait()
-
-	expectedBlockReward := consensus.NewDefaultBlockRewarder().BlockRewardAmount()
-	expectedPrice := types.NewAttoFILFromFIL(333)
-	expectedGasCost := big.NewInt(100)
-	expectedBalance := expectedBlockReward.Add(expectedPrice.MulBigInt(expectedGasCost))
-	newBalance := queryBalance(t, d, miningMinerOwnerAddr)
-	assert.Equal(t, expectedBalance.String(), newBalance.Sub(startingBalance).String())
+	t.Skip("new runtime")
```

Comment:
For the most part it means rewrite or delete.
Some of this tests were using features we n longer have because the runtime no longer needs. 
We need to revisit most of them and case by case analyze WHAT is being tested, and then determine how to test that in the new world.

Reply:
Oh looking at this again it was already skipped with a description of how and when to address on line 261